contracts/src/v0.8/dev/KeeperRegistryDev.sol

```diff
@@ -686,4 +713,20 @@ contract KeeperRegistryDev is
     if (msg.sender != owner() && msg.sender != s_registrar) revert OnlyCallableByOwnerOrRegistrar();
     _;
   }
+
+  /**
+   * @dev Reverts if called on a paused upkeep.
+   */
+  modifier onlyNonPausedUpkeep(uint256 id) {
+    if (s_upkeep[id].paused) revert OnlyNonPausedUpkeep();
```

Comment:
this line does a storage read, which is expensive, and the calling functions who use this modifier are also doing the same storage read. So we're paying for the same read twice. I think it will be cheaper and cleaner to remove both of these modifiers, and just do this check in the calling functions after `s_upkeep[id]` is read into memory.

Reply:
Done